y_lb.install

```diff
@@ -178,3 +178,31 @@ function y_lb_update_9010(&$sandbox) {
     'core.entity_view_display.node.landing_page_lb.default',
   ]);
 }
+
+/**
+ * Add metadata helper fields and enable meta tags
+ */
+function y_lb_update_9011(&$sandbox) {
+  $configs = [
+    'core.entity_form_display.node.landing_page_lb.default',
+    'core.entity_view_display.node.landing_page_lb.default',
+    'field.field.node.landing_page_lb.field_meta_description',
+    'field.field.node.landing_page_lb.field_meta_image',
+    'field.field.node.landing_page_lb.field_meta_tags',
+    'field.storage.node.field_meta_description',
+    'field.storage.node.field_meta_image',
+  ];
+
+  // Only import the metatag config if it's not already configured.
```

Comment:
Let's have a follow up ticket in order to cover
- when I enabled metatag after this module installed - system should make metatag a Priority 1 @froboy

Reply:
@podarok I don't totally understand here. This conditional was based on your [comment in slack](https://yusaslackinstance.slack.com/archives/C04M0FBJTGC/p1689700219261959?thread_ts=1689700115.747979&cid=C04M0FBJTGC):

> we need to check if metatag  is configured for the CT - do not display  data from these fields you are working on

We have the hard dependency on metatag now so if metatag is _not_ enabled when this update runs or the module is enabled then the config will definitely be imported. This was meant to deal with the case that you indicated - if someone has existing metatag configuration we should not override it - but maybe I misunderstood?

Reply:
ah, got it, clear now
disregard then my comment, thank you